### ISSUE TITLE
Feature Request: Option to auto play audio before selecting correct/incorrect in Anki Mode.

### ISSUE THREAD

**Issue by maboesanman** [davidsansome/tsurukame#735]
I use Anki mode and combine meaning + reading. I'd like the audio to play immediately upon showing the back of the card so I can compare what I said aloud to what I hear from the app.

**Comment by MichaelPKelley** [comment]
I agree, we see should get the audio at the same time we get the reading.

**Comment by BakaAdmin** [comment]
Agreed. This should be the behavior when auto play is enabled. Still doesn’t work that way in beta.
